Formula/sdl12-compat.rb

```diff
@@ -23,6 +24,7 @@ class Sdl12Compat < Formula
   def install
     system "cmake", "-S", ".", "-B", "build",
                     "-DSDL2_PATH=#{Formula["sdl2"].opt_prefix}",
+                    "-DCMAKE_INSTALL_RPATH=#{Formula["sdl2"].opt_lib}",
```

Comment:
This is probably ignored due to https://github.com/libsdl-org/sdl12-compat/blob/release-1.2.52/CMakeLists.txt#L18
```cmake
set(CMAKE_SKIP_RPATH TRUE)
```

Reply:
Thanks. Let's try `CMAKE_SHARED_LINKER_FLAGS`.